lib/ansible/plugins/inventory/netbox.py

```diff
@@ -198,6 +198,7 @@ def group_extractors(self):
             "device_roles": self.extract_device_role,
             "platforms": self.extract_platform,
             "device_types": self.extract_device_type,
+            "config_contexts": self.extract_config_contexts,
```

Comment:
Should it be `config_context`? There can be only one context per device.

Reply:
I agree. Will change it.